bin/spritesh.js

```diff
@@ -101,7 +102,10 @@ const filterFile = (file) => {
 
 const getSpriteContent = (contents) => {
   return '<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" '
-    + 'style="width: 0; height: 0; visibility: hidden; position: absolute;" aria-hidden="true">'
+    + (!program.styles
```

Comment:
my only suggestion would be to invert this condition.

```jsx
    + (program.styles
         ? 'style="width: 0; height: 0; visibility: hidden; position: absolute;"'
        : '')
```
imho, makes it more readable with less